components/authentication-framework/org.wso2.carbon.identity.application.authentication.framework/src/test/java/org/wso2/carbon/identity/application/authentication/framework/AuthenticationServiceTest.java

```diff
@@ -329,6 +353,12 @@ private String getFailureRedirectUrl(String sessionDataKey, String authenticator
                 "&authenticators=" + authenticators + "&authFailure=true&authFailureMsg=" + errorMsg;
     }
 
+    private String getPasswordExpiryUrl(String sessionDataKey, String errorMsg) {
+
+        return "/oauth2/authorize?sessionDataKey=" + sessionDataKey + "&passwordExpired=true&passwordExpiredMsg="
```

Comment:
can we use the existing constants here?

Reply:
This is now updated to use predefined constants.